_scripts/lesson1-3.R

```diff
@@ -109,3 +109,33 @@
     
 }
 
+# APPLICATION ---
+
+{
```

Comment:
You only need to have one curly bracket at the beginning and end, you do not have to put curly brackets around each code section of your script file.  You can and it hurts nothing to do it.  They are kind of like parenthesis in math equations -- you can put in as many as you want as long as you pair them off correctly.